amd_openvx/openvx/ago/ago_internal.h

```diff
@@ -229,10 +229,12 @@ typedef enum {
     ago_kernel_cmd_query_target_support       =  5,
 #if ENABLE_OPENCL
     ago_kernel_cmd_opencl_codegen             =  6,
+#elif ENABLE_HIP
+    ago_kernel_cmd_hip_codegen                =  6,
```

Comment:
this is removed in the MIVisionX ToT, please sync your code with ToT and remove this line.